src/schema/posts.ts

```diff
@@ -803,6 +806,28 @@ export const typeDefs = /* GraphQL */ `
     COLLECTION
   }
 
+  type UserPostEdge {
+    node: UserPost!
+
+    """
+    Used in \`before\` and \`after\` args
```

Comment:
Do we need the \\`?

Reply:
just copy pasted from other params

Reply:
probably not

Reply:
just because of same ticks are used inside the docs, removed them but it does not really make any difference